site/src/components/Navbar/MobileNavItems.tsx

```diff
@@ -270,6 +276,12 @@ export const MobileNavItems: VFC<MobileNavItemsProps> = ({ onClose }) => {
 
   return (
     <List>
+      {(asPath.includes("spec") || asPath.includes("docs")) && (
```

Comment:
Displays searchbar on mobile only if the path is in `spec` or `docs`

Reply:
This check might lead to false positives for other pages that happen to include the letters "spec" or "docs" - could you make it so that isn't possible?